server/src/mindustry/server/ServerControl.java

```diff
@@ -659,6 +659,16 @@ private void registerCommands(){
                 err("That IP/ID is not banned!");
             }
         });
+        
+        handler.register("pardon", "<ID>", "Pardons a votekicked player by ID and allows them to join again.", arg -> {
+            PlayerInfo info = netServer.admins.getInfo(arg[0]);
+            if(info != null){
+                info.lastKicked = 0;
+                info("Pardoned player: {0}", info.lastName);
+            } else{
```

Comment:
extra space

Reply:
I meant the space in `} else{` before the `else`.

Reply:
oh